### ISSUE TITLE
Additional Properties always null

### ISSUE THREAD

**Issue by youac** [mulesoft-labs/raml-for-jax-rs#351]
Hey,

After updating to 3.0.1, additionalProperties in the generated source models is always null. This causes a null pointer exception when a POST body contains any properties that were not defined in the RAML. 
This occurs when setting additionalProperties: true or false in the RAML

**Comment by jpbelang** [comment]
Something was fixed in 3.0.2.  Look at the snapshot (I''m looking at my code right now and it couldn't be null:

```java
@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonPropertyOrder("fingers")
public class ArmImpl implements Arm {
  @NotNull
  @JsonProperty(
      value = "fingers",
      defaultValue = "5"
  )
  private Integer fingers;

  @JsonIgnore
  private Map<String, Object> additionalProperties = new HashMap<String, Object>();
  @JsonAnyGetter
  public Map<String, Object> getAdditionalProperties() {
    return additionalProperties;
  }

  @JsonAnySetter
  public void setAdditionalProperties(String key, Object value) {
    this.additionalProperties.put(key, value);
  }
```

**Comment by youac** [comment]
Upon further testing, I realised that the problem was actually due to using a feature plugin to add lombok @builder annotation. Removing that fixed the issue, so it was never a problem from raml-for-jaxrs
